src/node/rpc/common_handler_registry.h

```diff
@@ -242,39 +253,32 @@ namespace ccf
           HTTP_STATUS_INTERNAL_SERVER_ERROR, "Unable to verify receipt");
       };
 
-      install(GeneralProcs::GET_COMMIT, json_adapter(get_commit), Read)
+      install_get(GeneralProcs::GET_COMMIT, json_adapter(get_commit), Read)
```

Comment:
As discussed before, my understand is that the `Read/Write` verb passed as the last argument to `install()` is now redundant and we could use the HTTP verb to forward or not (we can of course build a `.set...()` exception for `GET` handlers that end up writing to the KV).

In other words, it feels like there should only be one API for installing: `install(URI, lambda, HTTP_VERB)` so I am not too keen on having a specific `install_get()` and `install_post()`.